app/lib/three_scale/swagger/validator.rb

```diff
@@ -35,7 +31,7 @@ def valid_base_path?(record)
         # https://github.com/3scale/system/issues/4080
         return true if record.specification.base_path.blank?
```

Comment:
We should validate all base_path and not only the first one for OAS 3.0

Reply:
We still don't know what do with the multiple servers. Picking the first as _the_ base path was arbitrary.

Reply:
I'd rather not touch `base_path` in this iteration, but I've added validation of all servers anyway.